api/src/pcapi/connectors/typeform.py

```diff
@@ -200,7 +201,16 @@ def get_responses(self, form_id: str, page: int = 1, num_results: int = 1000) ->
                     case "email":
                         email = _strip(answer["email"])
                     case "choice":
-                        answers.append(TypeformAnswer(field_id=answer["field"]["id"], text=answer["choice"]["label"]))
+                        try:
+                            answers.append(
+                                TypeformAnswer(field_id=answer["field"]["id"], text=answer["choice"]["label"])
+                            )
+                        except KeyError:
+                            answers.append(
+                                TypeformAnswer(field_id=answer["field"]["id"], choice_id=answer["choice"]["id"])
+                            )
```

Comment:
Le "truc" qui marche à tous les coups ne serait-il pas ainsi ?
```suggestion
                        answers.append(
                            TypeformAnswer(field_id=answer["field"]["id"], choice_id=answer["choice"]["id"], text=answer["choice"].get("label"))
                        )
```